src/components/search/result/Filter.jsx

```diff
@@ -164,6 +165,24 @@ class Filter extends Component {
     });
   }
 
+  handleCheckBoxOnFocus(element) {
```

Comment:
To be consistent with the names of the other functions, this should be `handleCheckboxFocus` (lowercase b and remove On).